src/main/java/uk/gov/hmcts/reform/civil/model/CaseData.java

```diff
@@ -203,6 +204,7 @@ public class CaseData extends CaseDataParent implements MappableObject {
     private final DynamicList applicantSolicitor1PbaAccounts;
     private final ClaimTypeUnspec claimTypeUnSpec;
     private final ClaimType claimType;
+    private final HelpWithFeesDetails hwFeesDetails;
```

Comment:
Remove this if not required in your changes

Reply:
done